### ISSUE TITLE
Bug in adafruit_hid since library update

### ISSUE THREAD

**Issue by skyblaster** [wokwi/wokwi-circuitpython-libraries#2]
Test script can be found here: https://github.com/dbisu/pico-ducky/blob/main/duckyinpython.py

Line 6: 'from adafruit_hid.keyboard import Keyboard'

```
Traceback (most recent call last):
  File "code.py", line 6, in <module>
  File "adafruit_hid/__init__.py", line 21, in <module>
ImportError: no module named '__future__'
```

**Comment by urish** [comment]
Can you please share the link to the wokwi project that shows this problem?

**Comment by skyblaster** [comment]
https://wokwi.com/projects/357021888172298241

**Comment by urish** [comment]
Try using a newer CircuitPython version by changing diagram.json line 11 to read:

```json
      "attrs": { "env": "circuitpython-7.3.3" }
```

(you can also try 8.0.2)

**Comment by skyblaster** [comment]
Thanks! That works to clear the error.
I was just using the default link to create new projects here: https://wokwi.com/projects/new/circuitpython-pi-pico

**Comment by urish** [comment]
Yes, we're using an older version for that link since 7.x.x had performance issues with the Pico (adafruit/circuitpython#5341), so we had to use a specific version in order to get the simulation as fast as possible. This may have been solved in recent 8.x releases (by adafruit/circuitpython#7201), but I haven't had a chance to thoroughly test it yet. Once we're sure 8.x works correctly, we'll probably switch the new project template to use the latest 8.x release.
